skyllh/core/analysis_utils.py

```diff
@@ -109,18 +109,27 @@ def polynomial_fit(
     ns : float
     """
     (params, cov) = np.polyfit(ns, p, deg, w=p_weight, cov=True)
+    if(deg == 2 and params[0] > 0):
+        deg = 1
+        (params, cov) = np.polyfit(ns, p, deg, w=p_weight, cov=True)
 
     if(deg == 1):
         (a, b) = (params[0], params[1])
         ns = (p_thr - b)/a
 
```

Comment:
I would remove this blank line

Reply:
which one? The one between the two 'if' statements or before 'return ns'?